lib/Basics/errors.dat

```diff
@@ -251,7 +251,7 @@ ERROR_QUERY_FULLTEXT_INDEX_MISSING,1571,"no suitable fulltext index found for fu
 ERROR_QUERY_INVALID_DATE_VALUE,1572,"invalid date value","Will be raised when a value cannot be converted to a date."
 ERROR_QUERY_MULTI_MODIFY,1573,"multi-modify query","Will be raised when an AQL query contains more than one data-modifying operation."
 ERROR_QUERY_INVALID_AGGREGATE_EXPRESSION,1574,"invalid aggregate expression","Will be raised when an AQL query contains an invalid aggregate expression."
-ERROR_QUERY_COMPILE_TIME_OPTIONS,1575,"query options must be readable at query compile time","Will be raised when an AQL data-modification query contains options that cannot be figured out at query compile time."
+ERROR_QUERY_INVALID_OPTIONS,1575,"query options must be readable at query compile time","Will be raised when an AQL query contains OPTIONS that cannot be figured out at query compile time or when an unknown attribute is used inside OPTIONS."
```

Comment:
If `"query options must be readable at query compile time"` is what the user sees when the error is created, then we need a better message.

Reply:
Is it okay to just change an existing error code? Maybe we should just allocate a new one?